internal/pkg/aws/ecs/ecs.go

```diff
@@ -61,6 +63,34 @@ type TaskStatus struct {
 	StoppedReason string  `json:"stoppedReason"`
 }
 
+// HumanString returns the stringified TaskStatus struct with human readable format.
```

Comment:
What do you think of including a sample output in the comment for the reader to visualize the function?

Reply:
Good idea!